tests/intergation_test/test_gstreamer_output.py

```diff
@@ -234,7 +234,7 @@ def generate_reference_data() -> None:
     app.register_task(capture)
 
     def stop_callback() -> None:
-        app._handler(None, None)
```

Comment:
Call `stop()` directly because `Application#_handler` calls only `Application#stop`.

https://github.com/Idein/actfw-core/blob/81222b9a25b8236757470b869facd2a7f5d4be29/actfw_core/application.py#L36